frontend/src/stories/storyMocks.tsx

```diff
@@ -1,56 +1,190 @@
-import {
-  DefaultRequestBody,
-  graphql,
-  GraphQLHandler,
-  GraphQLRequest,
-  MockedRequest,
-  rest,
-  RestHandler,
-} from "msw";
+import { graphql, http, HttpResponse } from "msw";
 import {
   ApolloClient,
   HttpLink,
   InMemoryCache,
   ApolloProvider,
 } from "@apollo/client";
+import React from "react";
 
 import { exampleQuestionSet } from "../app/signUp/IdentityVerification/constants";
-import { UploadResponse, UploadSubmissionPage } from "../generated/graphql";
+import { UploadSubmissionPage } from "../generated/graphql";
 
-const BACKEND_URL = process.env.REACT_APP_BACKEND_URL;
+export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL;
 
-const mocks = {
```

Comment:
⚠️ Grouped by the mocks by HTTP GET, HTTP POST, MUTATIONS, QUERIES and within those groups they are in alphabetical order (hopefully 😄)